docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -324,8 +324,9 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
               // it can be placed below the admin label.
               $form['settings']['layout_builder_style_menu_orientation'] = [
                 '#weight' => -10,
-                  // We set #tree to FALSE because this value will override the
-                  // value from this will override the original in the form state.
+                  // We set #tree to FALSE because
+                  // this value will override the value from this
+                  // will override the original in the form state.
```

Comment:
This fixes the warning, but I'm also confused by the wording. Is it supposed to be "We set the #tree to FALSE because this value will override the original in the form state" (without the extra '...will  override the value from this...")?

Reply:
Looks like it originated here https://github.com/uiowa/uiowa/pull/4667. @GaryRidgway do you recall this?

Reply:
I don't recall it right now, so my context is fuzzy. cory seems to have wording that makes sense to me though